downstream/modules/platform/proc-controller-create-insights-credential.adoc

```diff
@@ -17,7 +38,39 @@ Use the following procedure to create a new credential for use with Red Hat Insi
 +
 image::ug-credential-types-popup-window-insights.png[Credentials insights pop up]
 +
-* *Username*: Enter a valid Red Hat Insights credential.
+* *Username*: Enter a valid Red Hat Insights credential. 
 * *Password*: Enter a valid Red Hat Insights credential.
-The Red Hat Insights credentials are the user's link:https://access.redhat.com/[Red Hat Customer Portal] account username and password.
++
+The {InsightsShort} credentials are the user's link:https://access.redhat.com/[Red Hat Customer Portal] account username and password.
++
+[NOTE]
+====
+Use the *Username* and *Password* fields for Basic authentication. 
+You can leave it blank if using *Client ID* and *Client secret*.
+====
++
+* *Client ID*: Enter the client ID you received when you created your service account. 
+* *Client secret*: Enter the client secret you received when you created your service account. 
+
 . Click btn:[Create credential].
++
+You can now use this credential in an xref:proc-controller-inv-source-insights[{InsightsShort}-sourced inventory] and xref:controller-create-insights-project[{InsightsShort} project].
+
+.Troubleshooting
+
+* If you receive a project sync failure, see the steps in xref:controller-remediate-insights-inventory[Remediating a Red Hat Insights inventory] and check your analytics logs.
+
+[IMPORTANT]
+====
+* You must recreate existing credentials and reassociate them with existing projects and inventory sources to support token-based authentication.
```

Comment:
this step is not needed - customers should be able to update existing AAP Credentials to use service account client id and client secret, and those AAP Credentials should still be associated with projects and inventory sources